src/Benchmarks/Benchmarks.csproj

```diff
@@ -10,7 +10,7 @@
     <UseWinmd Condition="'$(TargetFramework)' == 'netcoreapp3.1' And $(BenchmarkWinmdSupport) == true">true</UseWinmd>
     <ApplicationManifest Condition="$(UseWinmd) == true">Benchmarks.manifest</ApplicationManifest>
     <BenchmarkTargetFramework>$(TargetFramework)</BenchmarkTargetFramework>
-    <BenchmarkTargetFramework Condition="'$(BenchmarkTargetFramework)' == 'netcoreapp2.0'">netstandard2.0</BenchmarkTargetFramework>
+    <BenchmarkTargetFramework Condition="'$(BenchmarkTargetFramework)' != 'net5.0'">netstandard2.0</BenchmarkTargetFramework>
```

Comment:
@ujjwalchadha FYI, you might need to address these once these are merged.

Reply:
@j0shuams why was this change needed?

Reply:
@ujjwalchadha A few lines below this, we have a <Reference> node for the windows projection, and we use this property in the path. The windows projection only builds for netstandard2.0 and net5.0. So I was getting build failures when BenchmarkTargetFramework was netcoreapp3.1